src/fmu/sumo/utilities/scripts/sim2sumo.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+
+"""Upload data to Sumo from FMU."""
+
+import warnings
+import os
+import argparse
+import logging
+from pathlib import Path
+
+try:
+    from ert.shared.plugins.plugin_manager import hook_implementation  # type: ignore
+except ModuleNotFoundError:
+    from ert_shared.plugins.plugin_manager import hook_implementation  # type: ignore
+
+try:
+    from ert.shared.plugins.plugin_response import plugin_response  # type: ignore
+except ModuleNotFoundError:
+    from ert_shared.plugins.plugin_response import plugin_response  # type: ignore
+
+try:
+    from ert import ErtScript  # type: ignore
+except ModuleNotFoundError:
+    from res.job_queue import ErtScript  # type: ignore
+
+from fmu.sumo.utilities.sim2sumo import parse_args, upload_with_config
+
+LOGGER = logger = logging.getLogger(__name__)
+
+DESCRIPTION = """SIM2SUMO uploads results from reservoir simulators directly to sumo.
+Typically this is run per realization. The upload is controlled by a yaml config file
+
+SIM2SUMO is implemented both as a FORWARD_JOB and can be called only from this context
+ when running ERT."""
+
+EXAMPLES = """
+
+FOR full blown example add this to your ert config:
+
+FORWARD_MODEL SIM2SUMO(<CONFIG_PATH>= <your config location>, <SUMO_ENV>=<sumo env to upload to>)
```

Comment:
Is `<CONFIG_PATH>` a good name? I guess that can refer to several things, typically the `revision/ert/config` path. I would also guess that some FMU workflows have the variable `<CONFIG_PATH>` defined, but with a different meaning.

Reply:
It is now called S2S_CONF_PATH, so fixed